index.html

```diff
@@ -219,7 +221,11 @@ <h4>CONTACT US</h4>
       </div>
```

Comment:
- [ ] Instead of "Contact US" the form heading should change to "Let's chat!", please change it to the right heading, it's a requirement

![Screenshot_20220505_150822](https://user-images.githubusercontent.com/25227022/166954222-500bdbe0-9081-43c6-af57-b824d2b1cafa.png)